edit_course/views.py

```diff
@@ -159,7 +159,7 @@ def form_valid(self, form):
             self.object = form.save()
         except IntegrityError as e:
             messages.error(self.request,
-                _('Save failed: {error}').format(error=repr(e)))
+                _('Save failed: {error}.').format(error=repr(e)))
```

Comment:
`_("Failed to save {name} due to an error '{error}'.").format(name=self.model_name, error=e)`